submissions/xT0PERx/Memory pair game/transitions.js

```diff
@@ -0,0 +1,94 @@
+const images = [
+  'images/Alfa-romeo.jpeg',
+  'images/Ford.jpeg',
+  'images/Mazda.jpeg',
+  'images/Nissan.jpeg',
+  'images/Peugeot.jpeg',
+];
+
+const cardDeck = [...images, ...images];
+const duration = 600;
+let winCount = 0;
+let firstOpenCard = null;
+let isBlock = false;
+const pointsWon = images.length;
+const gameBoard = document.querySelector('.cards');
+
+function getSortList() {
+  cardDeck.sort(function () {
+    return 0.5 - Math.random();
+  });
+}
+
+const putImgsInCards = function () {
+  cardDeck.forEach(function (value) {
+    template = `
+        <div class="card">
+            <img class='front front-images' src='images/cardClosed.png' alt='front-card'>
+            <img class='back back__image' src='${value}' alt='back-card'>
+        </div>         
+      `;
+    gameBoard.insertAdjacentHTML('beforeEnd', template);
```

Comment:
This code modifies DOM in a loop, which is a heavy operation. Let's try to avoid it by creating a single element and then appending it to the DOM. Take a look at [Document Fragment API](https://developer.mozilla.org/en-US/docs/Web/API/DocumentFragment).